typescript/sdk/src/deploy/verify/ContractVerifier.ts

```diff
@@ -296,12 +296,6 @@ export class ContractVerifier extends BaseContractVerifier {
     verificationLogger: Logger = this.logger,
   ): Promise<ContractVerificationStatus> {
     try {
-      const metadata = this.multiProvider.tryGetChainMetadata(chain);
-      const rpcUrl = metadata?.rpcUrls[0].http ?? '';
-      if (rpcUrl.includes('localhost') || rpcUrl.includes('127.0.0.1')) {
-        verificationLogger.debug('Skipping verification for local endpoints');
-        return ContractVerificationStatus.Skipped;
-      }
```

Comment:
Was this supposed to be completely removed? I see that you created a new method below that does the same

Reply:
yeah this method is only used in the warpRead.getContractVerified() with the intention to skip checks, but i also needed it in the new ownerStatus.

moved a generalized version to the MultiProvider instead